src/Service/EdroneService.php

```diff
@@ -133,9 +251,26 @@ private function createSubscribeData(?string $firstName, string $email): array
         ];
     }
 
+    private function sendProduct(array $params): void
```

Comment:
Zastanawia mnie ta różnica pomiędzy 
`private function sendProduct(array $params): void`
A 
`private function sendPost(array $params): void`

Te same endpointy, to damo zastosowanie... Na pewno nie można obu wysłać tą samą metodą? Dodatkowo warto się zastanowić czy nie zaciągnąć tutaj do pracy Guzzle.